run_assistant.py

```diff
@@ -0,0 +1,107 @@
+import json
+import numpy as np
+import os
+import pyaudio
+import wave
+
+from aiy.board import Board, Led
+from deepspeech import Model, version
+from timeit import default_timer as timer
+
+from audio_response import *
+from rasa_api_call import *
```

Comment:
I think it makes code more readable if we just `import audio_response`, and then use `audio_response.method/func/whatever` explicitly. Same for `rasa_api_call`. These two modules only contain one class each, so perhaps it isn't a big deal in this case, especially since I'm sure we're doing this in other places as well.

Reply:
Thanks for the suggestion, @snekiam. I made the changes